exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/trace_exporter/__init__.py

```diff
@@ -159,8 +160,12 @@ def _translate_links(self, sdk_span: SDKSpan) -> None:
 
     def _translate_status(self, sdk_span: SDKSpan) -> None:
         if sdk_span.status is not None:
+            # TODO: Update this when the proto definitions are updated to include UNSET and ERROR
```

Comment:
Is this TODO required for GA? If so, may be create an issue and link it here?

Reply:
https://github.com/open-telemetry/opentelemetry-python/issues/1294